wiki/Game_Modes/External_Ports/T-Aiko!/Interface/en.md

```diff
@@ -0,0 +1,125 @@
+<!-- If possible, use a script to do this, mate. Pardon the terrible naming sense. ~RaikireHiuduo -->
```

Comment:
I thought tables and images don't mix (I've noted to **never** put images inside tables because peppy said to not do that).
A line break might work, but the way how it looks, the description could get confused with either the top or the bottom image (though, I assume the description is for the image above it).